packages/calcite-components/src/components/button/button.tsx

```diff
@@ -104,6 +104,11 @@ export class Button
   @Prop({ reflect: true })
   form: string;
 
+  /**
```

Comment:
For consistency, can you use the same [JSDoc from link](https://github.com/Esri/calcite-design-system/blob/93b2563a7e28b97d81bda66cbfa2ada5a8df9847/packages/calcite-components/src/components/link/link.tsx#L39-L43) for this property?

Reply:
Sure!